src/components/SourceItem/SourceItem.tsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable new-cap */
+import Plausible from 'plausible-tracker';
+import { Grid, Paper, Link, Box } from '@mui/material';
+import Typography from '../Typography';
+import React from 'react';
+import SupportIndicator from '../SupportIndicator/SupportIndicator';
+
+interface SourceProps {
+  claim: Claim;
+  /* The source supporting/rejecting its corresponding claim */
+  source: Source;
+  onFeedbackSourceRelevance?: OnFeedbackSourceRelevance;
+  onFeedbackSourceSupport?: OnFeedbackSourceSupport;
+}
+
+interface DisplayLinkProps {
+  /* The link displayed in the UI */
+  displayLink: string;
+  /* The full URL */
+  link: string;
+}
+
+/* A component to display the link to the article and its publishing date */
+const DisplayLink: React.FC<DisplayLinkProps> = ({ displayLink, link }) => {
+  const { trackEvent } = Plausible();
+
+  const onLinkClicked = () => {
+    trackEvent('Button: Link to source');
+  };
+
+  return (
+    <Grid
+      position="relative"
+      container
+      alignItems="center"
+      spacing={1}
+      justifyContent="center"
+      className="DisplayLink"
+    >
+      <Grid item sx={{ position: 'absolute' }}>
+        <img
+          height={50}
+          width={50}
```

Comment:
Favicons are pretty low resolution, maybe we should set the size to 32x32.

Reply:
Ok, should I also make a note in sigma about it? I went with the size from there

Reply:
Sure, add a comment there to let Petter know.